packages/governance/test/swingsetTests/committeeBinary/bootstrap.js

```diff
@@ -228,10 +228,16 @@ const makeBootstrap = (argv, cb, vatPowers) => async (vats, devices) => {
   const [testName] = argv;
   switch (testName) {
     case 'committeeBinaryStart':
-      committeeBinaryStart(zoe, voterCreator, timer, log, installations);
+      await committeeBinaryStart(zoe, voterCreator, timer, log, installations);
```

Comment:
I don't think `await` is doing anything helpful here. Wouldn't `void` be clearer?
[same below]

Reply:
It's preserving the stack trace if `committeeBinaryStart` fails. Without these awaits, `makeBootstrap` will complete as soon as the case functions are called. If they fail it will throw in Node as an unhandled promise rejection.

I think if they fail then `makeBootstrap` should also fail.